docs/docs/command-line-interface.mdx

```diff
@@ -225,7 +223,7 @@ and
 rasa test nlu
 ```
 
-Check out more details in
+You can find more details in
 [Evaluating an NLU Model](./testing-your-assistant.mdx#evaluating-an-nlu-model) and
 [Evaluating a Core Model](./testing-your-assistant.mdx#evaluating-a-core-model).
```

Comment:
```suggestion
[Evaluating a Core Model](./testing-your-assistant.mdx#evaluating-a-core-model).
```
This should get updated on that page and in this link to "Evaluating a Dialogue Model"

Also a comment for me i guess, since testing your assistant is my paage 😋